### ISSUE TITLE
Cannot use original column names in FeatureInfoTemplate once renamed

### ISSUE THREAD

**Issue by RacingTadpole** [TerriaJS/terriajs#2064]
This was implemented in the old UI, and maybe even in the new UI once, though I can't find it in the history.
Here's an example of it failing:
![image](https://cloud.githubusercontent.com/assets/1757858/18500244/331e1f8a-7a89-11e6-9541-d67b6451b707.png)

http://localhost:3003/#build/terriajs/test/init/test-tablestyle.json , Csv Column-specific -> Template with new names.

**Comment by RacingTadpole** [comment]
In the old UI it was done via:

```
// If property names were changed, let the template access the original property names too.
if (defined(result) && defined(result._terria_columnAliases)) {
    for (var i = 0; i < result._terria_columnAliases.length; i++) {
        var alias = result._terria_columnAliases[i];
        result[alias.id] = result[alias.name];
    }
```

**Comment by kring** [comment]
Fixed by #2066
